File: palindrome/palindrome.cs
using System;

class Program {
    static bool IsPalindrome(string str) {
        str = str.ToLower();
        char[] arr = str.ToCharArray();
        Array.Reverse(arr);
        return str == new string(arr);
    }
    
    static void Main() {
        Console.WriteLine(IsPalindrome("racecar"));
        Console.WriteLine(IsPalindrome("hello"));
    }
}

File: palindrome/palindrome.php
<?php
function isPalindrome($str) {
    $str = strtolower($str);
    return $str === strrev($str);
}

echo isPalindrome("racecar") ? "true" : "false";
echo "\n";
echo isPalindrome("hello") ? "true" : "false";
echo "\n";
?>
